hack/addon/readme/CSIOperator.md

```diff
@@ -31,6 +31,23 @@ CSIOperator 用于支持集群方便的使用存储资源，当前支持的存
 4. 在弹出的扩展组件列表里，滑动列表窗口找到 CSIOperator
 5. 单击【完成】进行安装
 
+### 镜像准备
+
+当前支持的存储插件包括 RBD、CephFS、TencentCBS 和 TencentCFS（测试中，暂不支持），
+针对不同的模式，依赖不同的 csi 镜像：
```

Comment:
please highlight here that the dependency image not in tkestack built-in registry but all in docker hub tkestack org, since it will cause the offline package too big